src/ajax.js

```diff
@@ -40,7 +40,7 @@
     };
 
     $private.XHRConnection = function XHRConnection( type, url, data ) {
-      var xhr = new XMLHttpRequest();
+      var xhr = new(XMLHttpRequest || ActiveXObject)('MSXML2.XMLHTTP.3.0');
```

Comment:
IMHO it`s not as if this is some crazy hacking, It`s just a fallback that is here to help the lib usage to be as broad as possible.